### DISCUSSION TITLE
Is it possible to update arbitrary files via regex?

### DISCUSSION THREAD

**Discussion by mueller-ma** [renovatebot/renovate#28798] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Is it possible to update files not related to dependencies with Renovate? For example when we have to update the file `CODEOWNERS` in many repositories:

````
* @JohnDoe
````

to 
````
* @JaneDoe
````

using something like 

````json
"customManagers": [
  {
    "customType": "regex",
    "fileMatch": [
      "CODEOWNERS"
    ],
    "matchStrings": [
      "@JohnDoe"
    ],
    "autoReplaceStringTemplate": "@JaneDoe",
  }
]
````

This json snippet doesn't work as there's no `packageName` or `packageNameTemplate`.

**Comment by rarkins** [comment]
It wasn't intended to work that way but maybe it might work with after trial and error experimentation
